framework/src/base/NonlinearSystem.C

```diff
@@ -240,6 +240,9 @@ void
 NonlinearSystem::setupStandardFiniteDifferencedPreconditioner()
 {
 #if LIBMESH_HAVE_PETSC
+  // Make sure that libMesh isn't going to override our preconditioner
+  _transient_sys.nonlinear_solver->jacobian = NULL;
```

Comment:
Since you have to fixup this PR, you may as well fix this too: `NULL` -> `nullptr`